org/postgresql/core/Parser.java

```diff
@@ -362,4 +430,257 @@ private static boolean subArraysEqual(final char[] arr,
     
         return true;
     }
+
+    /**
+     * Contains parse flags from {@link #modifyJdbcCall(String, boolean, int, int)}.
+     * Originally {@link #modifyJdbcCall(String, boolean, int, int)} was located in {@link org.postgresql.jdbc2.AbstractJdbc2Statement},
+     * however it was moved out to avoid parse on each prepareCall.
+     */
+    public static class JdbcCallParseInfo {
```

Comment:
Should this class be public?  Also, I would strongly recommend making these fields `final`.